esp32c3-hal/ld/db-riscv-link.x

```diff
@@ -110,6 +110,16 @@ SECTIONS
     _ebss = .;
   } > REGION_BSS
 
+  /* ### .uninit */
+  .uninit (NOLOAD) : ALIGN(4)
+  {
+    . = ALIGN(4);
+    __suninit = .;
+    *(.uninit .uninit.*);
+    . = ALIGN(4);
+    __euninit = .;
+  } > REGION_BSS
```

Comment:
nitpicking but maybe we should define an alias in `db-esp32c3-memory.x` like it's done for other regions - doesn't change anything but would look more consistent

Reply:
I merely reposted the original solution from @9names, I have no experience with linker scripts so I'm not sure how to do that 😅

Feel free to modify the pr though!